tests/files/features/.lagoon.yml.test

```diff
@@ -0,0 +1,53 @@
+docker-compose-yaml: docker-compose.yml
+
+environment_variables:
+  git_sha: 'true'
+
+environments:
+  branch/routes:
+    routes:
+    - node:
+      - customdomain-will-be-main-domain.com
+      - customdomain-will-be-not-be-main-domain.com
+  branch/cronjob:
+    cronjobs:
```

Comment:
can we remove all cronjobs here? with that we also make sure that mulitple cronjobs can also be removed

Reply:
It won't be as easy as simply removing the cronjobs, due to the way the node test works. It just dumps the contents of the `files/cron.txt` into its HTTP response. Let me sleep on it and I'm sure I can think of something to test both of these cases.

Reply:
Done in 7e4bba47